index.html

```diff
@@ -17,7 +17,7 @@
     <div class="container py-4">
         <header class="pb-3 mb-4 border-bottom">
             <a href="/" class="d-flex align-items-center text-dark text-decoration-none">
-                <span class="fs-4">Open Website Alliance</span>
+                <img class="col-md-1" src="images/logos/OWA logo.png" alt="Open Website Alliance">
```

Comment:
I suggest we still keep the organization name in, but in a muted color. I also added an image height for better appearance on mobile screens.

```suggestion
                <img class="me-3" src="images/logos/OWA logo.png" height="80" alt="Open Website Alliance">
                <span class="fs-5 text-muted">Open Website Alliance</span>
```